README.md

```diff
@@ -31,12 +31,20 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your
+(open source) software dependencies in a strategic, safe and efficient manner.
```

Comment:
should we append: "...on an organization-wide scale" to the end of line 35?

Reply:
No, like to keep it short & simple - rest of the doc update will make it clear ORT is great for orgs.